spanner/jdbc/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <properties>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+  </properties>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.11</version>
+  </parent>
+
+  <groupId>com.example.spanner</groupId>
+  <artifactId>cloud-spanner-jdbc</artifactId>
+  <name>Cloud Spanner with JDBC Code Sample</name>
+  
+  <!-- [START spanner_jdbc_dependencies] -->
+  <dependencies>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-api</artifactId>
+      <version>1.27.2</version>
+    </dependency>
+    <!-- The Spanner JDBC driver dependency -->
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-spanner-jdbc</artifactId>
+      <version>1.13.1-SNAPSHOT</version>
```

Comment:
Don't use -SNAPSHOT's

Reply:
I'll change that as soon as a version of the JDBC driver with support for `QueryOptions` has been released. Until then this PR won't build anyways, so it can't be merged by accident either.